scripts/src/javatests/org/oppia/android/scripts/regex/RegexPatternValidationCheckTest.kt

```diff
@@ -207,7 +211,8 @@ class RegexPatternValidationCheckTest {
       """
       File name/path violation: $activitiesPlacementErrorMessage
       - data/src/main/TestActivity.kt
-
+      
+      data/src/main/TestActivity.kt: $doesNotContainScreenName
```

Comment:
For this & other tests, can we update these to avoid including these extra errors? They seem to be "bleeding into" some of these other tests, and they shouldn't be since tests should be verifying isolated behaviors.

Reply:
done.